### ISSUE TITLE
Dropping levels, camera scrolling

### ISSUE THREAD

**Issue by ace-dent** [Press-Play-On-Tape/PrinceOfArabia#140]
Tested with v1.01 Beta 008, Sim.

- Playing L7. There is a stage where you must drop 1 or 2 levels, and pull yourself up.
- The second (lower) level scrolls smoothly -- with the camera tracking the player (down and then up).
 ![2ndDropScrolling](https://user-images.githubusercontent.com/473224/227811930-992c8e6f-e46b-492e-be86-4cbc1f0b7b98.gif)
- BUT the first (upper) level looks jerky when it scrolls/ the background tiles seem to jump?
![1stDrop](https://user-images.githubusercontent.com/473224/227812014-76a35bb7-a583-4164-985a-4bdb80bc5237.gif)

**Comment by filmote** [comment]
Yeah .. I am not going to be fixing this as it is fundamental to the whole display of the level.  I could make the first one scroll nicer than the second by removing a top layer of bricks from the level.  But either way, one will be good and one will be bad.

**Comment by ace-dent** [comment]
I didn't realise this is fundamental to the rendering system (haven't noticed it elsewhere).
If it cannot be fixed, please feel free to Close.

**Comment by filmote** [comment]
In the real game, you can see the 3 levels / platforms of each 'room' without the screen scrolling.  On the Arduboy, I have to scroll up and down to make the screen work and have the graphics large enough that they look OK.

As there are three levels, I have made it so that it will scroll as you move between levels but will always show you the level you just came from.  It makes sense most times but there are situations when (say) you move from level 1 to 2 and want to continue to level 3 but until you start moving down, you cannot see the ground and cannot always see traps.  

So why not always show the level you are on and two half levels above and below?  Just as bad, as you still would not be able to see the ground below.  When standing on level2, I do not know if your next move is up or down :)

I guess that is what I meant by fundamental - attempting to change the scrolling would impact the game play overall.
